desktop/extensions-sdk/guides/security.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Security
+description: Aspects of the security model of extensions
+keywords: Docker, extensions, sdk, security
+---
+
+## Extension capabilities
+
+Extensions can provide various optional parts: a user interface (in html/javascript), a backend part (running as a container), and executables deployed on the host machine.
+
+Extensions are executed with the same permissions as the Docker Desktop user and extension capabilities include: running any docker commands (including running containers and mounting folders), running extension binaries, and accessing files on your machine.
+
+The Extensions SDK provides a set of JavaScript APIs to invoke commands or execute these binaries from the extension UI code. Extensions can also provide a backend part that will start a long-lived running container in the background.
+
+Learn more in the [architecture section](https://docs.docker.com/desktop/extensions-sdk/architecture/)
+
+Extensions code is loaded in Docker Desktop UI electron app. The extension UI code is subject to some constraints for security reasons, described in the following paragraphs.
+
+## Sandboxing
+
+Extension UI code is rendered in a sandboxed environment and does not have a node.js environment initialized, nor direct access to the electron APIs.
+The extension UI code cannot perform privileged tasks (such as making changes to the system, or spawning subprocesses) except by using the SDK APIs provided with the extension framework.
+It can also perform interactions with Docker Desktop (such as navigating to various places in the Dashboard) only through the extension SDK APIs.
```

Comment:
```suggestion
It can also perform interactions with Docker Desktop, such as navigating to various places in the Dashboard, only through the extension SDK APIs.
```